Update 50.discussion.md (#600)
greenelab/covid19-review: content/50.discussion.md

#### Access to and Representation in Clinical Trials

Scientific and medical research is broadly shaped by a number of biases.
Bias in medicine can be characterized as when a physician and/or patient “others” an individual in a healthcare setting.
Some concerns of medical bias are how clinical trials recruit participants and operate.
The objective is to synthesize literature on how these clinical trial biases affect Covid-19 outcomes. 
 
While researchers aim to quickly yet effectively develop a vaccine for Covid-19 [maybe mention some of the research mentioned in the paper], there needs to be global representation in clinical trials to ensure equality.
The benefits and burdens of clinical trials need to be equally distributed to ensure no influx of health disparities in the response.

> The Wealth Health Organization (WHO) Director‐General Tedros Adhanom Ghebreyesus stated his condemnation of utilizing low and middle income countries as test subjects for clinical trials, yet having highly developed countries as the majority of clinical trial representation is also not the answer [@doi:10.1002/eahr.500055].

Reviewer: ```suggestion
The WHO Director‐General Tedros Adhanom Ghebreyesus stated his condemnation of utilizing low and middle income countries as test subjects for clinical trials, yet having highly developed countries as the majority of clinical trial representation is also not the answer [@doi:10.1002/eahr.500055].
```

Figure X showcases two choropleths detailing Covid-19 clinical trial recruitment by country.
The first choropleth only recruits from a singular country while the second choropleth recruits from multiple countries.

> In the first choropleth, China, the United States, and France are among the countries with the most clinical trial recruiting.

Reviewer: ```suggestion
China, the United States, and France are among the countries with the most clinical trial recruiting for trials with single-country enrollment.
```

However, in regards to recruiting multiple countries, only the United States and some European countries have made an effort to include other countries in clinical trials.
In addition, many countries have little to no clinical trial recruiting, which may create inconclusivity in the data from clinical trials.
Especially with Covid-19, equitable access to therapeutics and vaccines have been at the forefront, yet figures like the choropleths show the lack of transparency.
Thus, singular governments might prioritize their people with vaccines, causing unequal health outcomes [@doi:10.1001/jama.2020.6641].
Especially, with the closing of borders, keeping resources, and exiling foreigners [continue]
 

> When there is equal opportunity to participate in clinical trials, there is an insignificant difference in participation between racial groups [@doi:10.1002/cncr.28483].

Author: I agree. It sounds more clear.

However, within the United States, racial minorities, such as, African-Americans, have disproportionately lower representation in clinical trials.

> Similarly, African-Americans are the most affected racial demographic by Covid-19.

Author: I see what you are saying. my statement was rather general, so it could be misconstrued. I agree with the changes.

Some studies suggest communication between physicians and patients impacts whether or not a physician offers a patient participation in a clinical trial.
The researchers utilized a linguistic analysis to assess mean word count of  phrases like voluntary participation, clinical trial, and etc.
The data states that the mean word count of the entire visit is 1.5 more for white patients in comparisons to Black patients.
In addition, a figure shows that the greatest disparity between white and Black patients is discussion on risks with over 2 times as many words spoken with white patients than that of Black patients [@doi:10.1002/cncr.28483].
Thus, it is likely that Covid-19 clinical trial information is not being equally discussed with Black patients. 
 

> Some plausible reasons for the disparity in communication between physicians and patients could be a lack of awareness and education, mistrust in healthcare professionals, and lack of health insurance [@doi:10.1002/cncr.28483].

Reviewer: ```suggestion
```
I moved this up! I think I changed the context and made it a bit more politically aggressive, though, so let me know if you don't like that!

Author: it flowed really well in the new location

 
Global representation in clinical trials is important to ensuring that experimental treatments are available equally to COVID-19 patients who may need them.
The advantage to a patient of participation in a clinical trial is that they may receive an experimental treatment they would not have been able to access otherwise.
The potential downsides of participation include that the efficacy and side effects of such treatments are often poorly characterized and that patients who enroll in clinical trials will in some cases run the risk of being assigned to a placebo condition where they do not receive the treatment but miss out on opportunities to receive other treatments.
The benefits and burdens of clinical trials therefore need to be weighed carefully to ensure that they don't reinforce existing health disparities.
The WHO Director‐General Tedros Adhanom Ghebreyesus stated his condemnation of utilizing low and middle income countries as test subjects for clinical trials, yet having highly developed countries as the majority of clinical trial representation is also not the answer [@doi:10.1002/eahr.500055].

> Figure @fig:ebm-trials-map showcases two choropleths detailing COVID-19 clinical trial recruitment by country.

Reviewer 2: ```suggestion
Figure @fig:ebm-map showcases two choropleths detailing COVID-19 clinical trial recruitment by country.
```

China, the United States, and France are among the countries with the most clinical trial recruiting for trials with single-country enrollment.
Many countries have little to no clinical trial recruiting, with the continents of Africa and South America much less represented than Asia, Europe, and North America.
Trials that recruit across multiple countries do appear to broaden geographic representation, but these trials seem to be heavily dominated by the United States and European Union.
Dual concerns arise from this skewed geographic representation in clinical trial recruitment: first, it suggests that treatments such as remdesivir that are promising but primarily available to clinical trial participants are unlikely to be accessible by people in many countries, and second, it raises the concern that the findings of clinical trials will be based on participants from many of the wealthiest countries, which may lead to ambiguity in whether the findings can be extrapolated to COVID-19 patients elsewhere.

> Especially with the global nature of COVID-19, equitable access to therapeutics and vaccines has been a concern at the forefront of many discussions about policy (e.g., [@doi:10/ggq7mf], yet data like that shown in Figure @fig:ebm-trials-map demonstrates that accessibility is likely to be a significant issue.

Reviewer 2: ```suggestion
Especially with the global nature of COVID-19, equitable access to therapeutics and vaccines has been a concern at the forefront of many discussions about policy (e.g., [@doi:10/ggq7mf], yet data like that shown in Figure @fig:ebm-map demonstrates that accessibility is likely to be a significant issue.
```

Another concern with the heterogeneous international distribution of clinical trials is that the governments of countries leading these clinical trials might prioritize their own populations once vaccines are developed, causing unequal health outcomes [@doi:10.1001/jama.2020.6641].
Additionally, even within a single state in the United States (Maryland), geography was found to influence the likelihood of being recruited into or enrolled in a clinical trial, with patients in under-served rural areas less likely to enroll [@doi:10.1016/j.cdp.2005.12.001].
Thus, geography both on the global and local levels may influence when treatments and vaccines are available and who is able to access them.
Efforts such as the African Union's efforts to coordinate and promote vaccine development [@doi:10/fgzk] are therefore critical to promoting equity in the COVID-19 response.
 
 ![
**Geographic distribution of COVID-19 clinical trials.**
The density of clinical trials is reported at the country level.

> As of {{ebm_date_pretty}}, there are {{ebm_trials}} in the University of Oxford Evidence-Based Medicine Data Lab's COVID-19 TrialsTracker [@doi:10.5281/zenodo.3732709].

Reviewer 2: ```suggestion
As of {{ebm_date_pretty}}, there are {{ebm_trials}} trials in the University of Oxford Evidence-Based Medicine Data Lab's COVID-19 TrialsTracker [@doi:10.5281/zenodo.3732709].
```

The top figure demonstrates the density of trials recruiting only from a singular country, while the bottom shows the distribution of recruitment for trials that involve more than one country.

> ]({{ebm_trials_map}} "COVID-19 clinical trials"){#fig:ebm-map secno=1}

Reviewer 2: ```suggestion
]({{ebm_map_figure}} "COVID-19 clinical trials"){#fig:ebm-map secno=1}

```

Because racism is a common factor in both, negative patient experiences with medical professionals are likely to compound other issues of systemic inequality, such as a lack of access to adequate care, a lack of insurance, or increased exposure to SARS-CoV-2 [@doi:10.1542/peds.2020-003657].
Furthermore, the experience of being othered is not only expected to impact patients' trust in and comfort with their provider, but also may directly impact whether or not the patient is offered the opportunity to participate in a clinical trial at all.
Some studies suggest communication between physicians and patients impacts whether or not a physician offers a patient participation in a clinical trial.
For example, researchers utilized a linguistic analysis to assess mean word count of phrases related to clinical trial enrollment, such as voluntary participation, clinical trial, etc. [@doi:10.1002/cncr.28483].

> The data indicated that the mean word count of the entire visit was 1.5 more for white patients in comparison to Black patients.

Reviewer 3: 1.5 times more???

> The data indicated that the mean word count of the entire visit was 1.5 more for white patients in comparison to Black patients.

Reviewer 3: ```suggestion
The data indicated that the mean word count of the entire visit was 1.5 times more for white patients in comparison to Black patients.
```

> The data indicated that the mean word count of the entire visit was 1.5 more for white patients in comparison to Black patients.

Reviewer 2: ```suggestion
The data indicated that the mean word count of the entire visit was 1.5 time more for white patients in comparison to Black patients.
```

I agree with @RLordan's suggestion

In addition, the greatest disparity between white and Black patients' experience was the discussion of risks, with over 2 times as many risk-related words spoken with white patients than Black patients [@doi:10.1002/cncr.28483].

> Thus, it is likely that Covid-19 clinical trial information is not being equally discussed with Black patients. 

Reviewer 2: ```suggestion
Thus, it is likely that COVID-19 clinical trial information is not being equally discussed with Black patients. 
```

 These discrepancies are especially concerning given that many COVID-19 treatments are being or are considered being made available to patients prior to FDA approval through Emergency Use Authorizations.
In the past, African-Americans have been over-represented relative to national demographics in use of the FDA's Exception From Informed Consent (EFIC) pathway [@doi:10.1377/hlthaff.2018.0501].
Through this pathway, people who are incapacitated can receive an experimental treatment even if they are not able to consent and there is not sufficient time to seek approval from an authorized representative.
This pathway presents concerns, however, when it is considered in the context of a long history of systematic abuses in medical experimentation where informed consent was not obtained from people of color, such as the Tuskegee syphilis experiments [@doi:10.1067/mem.2003.17].
While the goal of EFIC approval is to provide treatment to patients who urgently need it, the combination of the ongoing legacy of racism in medicine renders this trend concerning.

> With COVID-19, efforts to prioritize people who suffer from systemic racism are often designed with the goal of righting some of these inequalities (e.g., [@url:https://www.businessinsider.com/cdc-official-considering-giving-covid-19-vaccine-most-vulnerable-first-2020-10], but particular attention to informed consent will be imperative in ensuring these trials are ethical given that the benefits and risks of emerging treatments are still poorly characterized.

Reviewer 2: ```suggestion
With COVID-19, efforts to prioritize people who suffer from systemic racism are often designed with the goal of righting some of these inequalities (e.g., [@url:https://www.businessinsider.com/cdc-official-considering-giving-covid-19-vaccine-most-vulnerable-first-2020-10]), but particular attention to informed consent will be imperative in ensuring these trials are ethical given that the benefits and risks of emerging treatments are still poorly characterized.
```

Making a substantial effort to run inclusive clinical trials is also important because of the possibility that racism could impact how a patient responds to a treatment.
For example, as discussed above, dexamethasone has been identified as a promising treatment for patients experiencing cytokine release syndrome, but the mechanism of action is tied to the stress response.

> A study from 2005 reported that Black asthma patients showed reduced responsiveness to dexamethsone in comparison to white patients and suggested Black patients might therefore require higher doses of the drug [@doi:10.1378/chest.127.2.571].

Reviewer: ```suggestion
A study from 2005 reported that Black asthma patients showed reduced responsiveness to dexamethasone in comparison to white patients and suggested Black patients might therefore require higher doses of the drug [@doi:10.1378/chest.127.2.571].
```

#### Access to and Representation in Clinical Trials

Experimental treatments are often made available to patients primarily or even exclusively through clinical trials.
The advantage of this approach is that clinical trials are designed to collect rigorous data about the effects of a treatment on patients.

> The disadvantage, however, is that access to clinical trials is not equal among all people who suffer from a disease.

Reviewer 4: ```suggestion
The disadvantage is that access to clinical trials is not equal among all people who suffer from a disease.
```
word felt extra since contrast starts the sentence. not strongly tied to this change

The benefits and burdens of clinical trials therefore need to be weighed carefully to ensure that they don't reinforce existing health disparities.
The WHO Director‐General Tedros Adhanom Ghebreyesus stated his condemnation of utilizing low and middle income countries as test subjects for clinical trials, yet having highly developed countries as the majority of clinical trial representation is also not the answer [@doi:10.1002/eahr.500055].
Figure @fig:ebm-map showcases two choropleths detailing COVID-19 clinical trial recruitment by country.
China, the United States, and France are among the countries with the most clinical trial recruiting for trials with single-country enrollment.
Many countries have little to no clinical trial recruiting, with the continents of Africa and South America much less represented than Asia, Europe, and North America.
Trials that recruit across multiple countries do appear to broaden geographic representation, but these trials seem to be heavily dominated by the United States and European Union.

> Dual concerns arise from this skewed geographic representation in clinical trial recruitment: first, it suggests that treatments such as remdesivir that are promising but primarily available to clinical trial participants are unlikely to be accessible by people in many countries, and second, it raises the concern that the findings of clinical trials will be based on participants from many of the wealthiest countries, which may lead to ambiguity in whether the findings can be extrapolated to COVID-19 patients elsewhere.

Reviewer: ```suggestion
**Geographic distribution of COVID-19 clinical trials.**
The density of clinical trials is reported at the country level.
As of {{ebm_date_pretty}}, there are {{ebm_trials}} trials in the University of Oxford Evidence-Based Medicine Data Lab's COVID-19 TrialsTracker [@doi:10.5281/zenodo.3732709].
The top figure demonstrates the density of trials recruiting only from a singular country, while the bottom shows the distribution of recruitment for trials that involve more than one country.
]({{ebm_map_figure}} "COVID-19 clinical trials"){#fig:ebm-map secno=1}

A few different concerns arise from this skewed geographic representation in clinical trial recruitment.
First, treatments such as remdesivir that are promising but primarily available to clinical trial participants are unlikely to be accessible by people in many countries.
Second, it raises the concern that the findings of clinical trials will be based on participants from many of the wealthiest countries, which may lead to ambiguity in whether the findings can be extrapolated to COVID-19 patients elsewhere.
```

> Dual concerns arise from this skewed geographic representation in clinical trial recruitment: first, it suggests that treatments such as remdesivir that are promising but primarily available to clinical trial participants are unlikely to be accessible by people in many countries, and second, it raises the concern that the findings of clinical trials will be based on participants from many of the wealthiest countries, which may lead to ambiguity in whether the findings can be extrapolated to COVID-19 patients elsewhere.

Reviewer 4: What do you think about breaking up this paragraph somewhere around here.

Especially with the global nature of COVID-19, equitable access to therapeutics and vaccines has been a concern at the forefront of many discussions about policy (e.g., [@doi:10/ggq7mf], yet data like that shown in Figure @fig:ebm-map demonstrates that accessibility is likely to be a significant issue.
Another concern with the heterogeneous international distribution of clinical trials is that the governments of countries leading these clinical trials might prioritize their own populations once vaccines are developed, causing unequal health outcomes [@doi:10.1001/jama.2020.6641].
Additionally, even within a single state in the United States (Maryland), geography was found to influence the likelihood of being recruited into or enrolled in a clinical trial, with patients in under-served rural areas less likely to enroll [@doi:10.1016/j.cdp.2005.12.001].
Thus, geography both on the global and local levels may influence when treatments and vaccines are available and who is able to access them.
Efforts such as the African Union's efforts to coordinate and promote vaccine development [@doi:10/fgzk] are therefore critical to promoting equity in the COVID-19 response.
 

>  ![
> **Geographic distribution of COVID-19 clinical trials.**
> The density of clinical trials is reported at the country level.
> As of {{ebm_date_pretty}}, there are {{ebm_trials}} trials in the University of Oxford Evidence-Based Medicine Data Lab's COVID-19 TrialsTracker [@doi:10.5281/zenodo.3732709].
> The top figure demonstrates the density of trials recruiting only from a singular country, while the bottom shows the distribution of recruitment for trials that involve more than one country.
> ]({{ebm_map_figure}} "COVID-19 clinical trials"){#fig:ebm-map secno=1}

Reviewer: ```suggestion
```
Moved up

Even when patients are located within the geographic recruitment area of clinical trials, however, there can still be demographic inequalities in enrollment.
When efforts are made to ensure equal opportunity to participate in clinical trials, there is no significant difference in participation among racial/ethnic groups [@doi:10.1002/cncr.28483].
However, within the United States, real clinical trial recruitment numbers have indicated for many years that racial minorities, especially African-Americans, tend to be under-represented (e.g., [@doi:10.1001/jama.291.22.2720; @doi:10.1245/s10434-007-9500-y; @doi:10.1089/jwh.2010.2469; @doi:10.1007/s11926-018-0728-2]).
This trend is especially concerning given the disproportionate impact of COVID-19 on African-Americans.
Early evidence suggests that the proportion of Black, Latinx, and Native American participants in clinical trials for drugs such as remdesivir is much lower than the representation of these groups among COVID-19 patients [@doi:10.1056/NEJMp2021971].

> One proposed explanation for differences among racial and ethnic groups in clinical trial enrollment refers to different experiences in healthcare settings.

Reviewer: ```suggestion

One proposed explanation for differences among racial and ethnic groups in clinical trial enrollment refers to different experiences in healthcare settings.
```

While some plausible reasons for the disparity in communication between physicians and patients could be a lack of awareness and education, mistrust in healthcare professionals, and a lack of health insurance [@doi:10.1002/cncr.28483], a major concern is that patients from certain racial and ethnic groups are marginalized even while seeking healthcare.

> In the United States, many patients experience “othering" from physicians and other medical professionals due to their race or other external characteristics such as gender (e.g., [@doi:10.1207/S15327027HC1602_7]).

Reviewer 4: ```suggestion
In the United States, many patients experience "othering" from physicians and other medical professionals due to their race or other external characteristics such as gender (e.g., [@doi:10.1207/S15327027HC1602_7]).
```

Reviewer 4: I think you might also want to add a paragraph break below this sentence.

Furthermore, the experience of being othered is not only expected to impact patients' trust in and comfort with their provider, but also may directly impact whether or not the patient is offered the opportunity to participate in a clinical trial at all.
Some studies suggest communication between physicians and patients impacts whether or not a physician offers a patient participation in a clinical trial.
For example, researchers utilized a linguistic analysis to assess mean word count of phrases related to clinical trial enrollment, such as voluntary participation, clinical trial, etc. [@doi:10.1002/cncr.28483].
The data indicated that the mean word count of the entire visit was 1.5 times more for white patients in comparison to Black patients.
In addition, the greatest disparity between white and Black patients' experience was the discussion of risks, with over 2 times as many risk-related words spoken with white patients than Black patients [@doi:10.1002/cncr.28483].

> Thus, it is likely that COVID-19 clinical trial information is not being equally discussed with Black patients. 

Reviewer 4: This might be better framed as: if it's like other trials, then...

The paper above pre-dates COVID-19.

Reviewer: ```suggestion
The trends observed for other clinical trials raise the concern that COVID-19 clinical trial information may not be discussed as thoroughly or as often with Black patients compared to white patients. 

```
